src/xdsl/xdsl_opt_main.py

```diff
@@ -133,6 +159,14 @@ def register_all_arguments(self, arg_parser: argparse.ArgumentParser):
                                 action='store_true',
                                 help="Prints the content of a triggered "
                                 "exception and exits with code 0")
+        arg_parser.add_argument(
+            "--verbose-trace",
+            type=int,
+            nargs=2,
+            required=False,
+            help="Prints the verbose traceback"
+            "First argument: number of lines of code for the snippet. "
+            "Second argument: number of output frames ")
```

Comment:
Also, this message should somehow include the dependency on `--verify-diagnostics`.